index.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Group } from 'ol/layer';
+import BaseLayer from 'ol/layer/Base';
+import Control, { Options as ControlOptions } from 'ol/control/Control';
+import PluggableMap from 'ol/PluggableMap';
+
+type GroupSelectStyle = 'none' | 'children' | 'group';
+
+export interface Options extends ControlOptions {
+    activationMode?: string;
+    collapseLabel?: string;
+    label?: string;
+    tipLabel?: string;
+    groupSelectStyle?: GroupSelectStyle;
+    reverse?: boolean
+}
+
+export default class LayerSwitcher extends Control {
+    constructor(options?: Options);
+    static isBaseGroup(lyr: BaseLayer): boolean;
+    static renderPanel(map: PluggableMap, panel: Element, options?: {groupSelectStyle?: GroupSelectStyle; reverse?: boolean;}): void;
+    static setGroupVisibility(map: PluggableMap): any[];
```

Comment:
`setGroupVisibility` doesn't return so should be `void`? Same for `setChidVisibility`.

Reply:
if it doesn't return then yes, it should be `void`

Reply:
i pushed a change for both static methods